src/georaster-layer-for-leaflet.ts

```diff
@@ -308,11 +328,12 @@ const GeoRasterLayer = L.GridLayer.extend({
                 values = tileRasters.map(band => band[h][w]);
               } else if (rasters) {
                 // get value from array with data for entire raster
-                values = rasters.map(band => {
+                values = rasters.map((band: Georaster) => {
+                  // @ts-ignore it works but needs validation. Current error: Type 'null' cannot be used as an index type
                   return band[yInRasterPixels][xInRasterPixels];
                 });
               } else {
-                done && done("no rasters are available for, so skipping value generation");
+                done && done(Error("no rasters are available for, so skipping value generation"));
```

Comment:
cool.  I thought you needed a `new` before Error, but researched it and I was totally wrong. Thanks for showing me this! 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error#function_call_or_new_construction

Reply:
Ah, I didn't have that sort of clarity. Thanks for the link.